nixos/modules/services/video/frigate.nix

```diff
@@ -94,6 +103,15 @@ let
     proxy_connect_timeout 360;
   '';
 
+  # Discover configured detectors for acceleration support
+  detectors = attrValues cfg.settings.detectors or {};
+  withCoralUSB = any (d: d.type == "edgetpu" && hasPrefix "usb" d.device or "") detectors;
+  withCoralPCI =  any (d: d.type == "edgetpu" && hasPrefix "pci" d.device or "") detectors;
+  withCoral = withCoralPCI || withCoralUSB;
+
+  # Provide ffmpeg-full for NVIDIA hardware acceleration
+  ffmpegArgs = cfg.settings.ffmpeg.hwaccel_args or "";
+  ffmpeg' = if match "/nvidia/" ffmpegArgs != null then pkgs.ffmpeg-full else pkgs.ffmpeg-headless;
```

Comment:
cudaLLVM.

Reply:
https://github.com/NixOS/nixpkgs/pull/354952 now fixes this.

Reply:
This gets backported, so unless the uplift also gets backported I can only make use of this later.

Reply:
Theoretically, we could backport. I don't really care what happens on 24.11 w.r.t. ffmpeg-full though, so this would also be fine. I'd just revert this when the time comes to remove ffmpeg-full tree-wide.